index.d.ts

```diff
@@ -6,6 +6,11 @@ declare module 'connected-react-router' {
   interface ConnectedRouterProps {
     history: History
   }
+  
+  export interface LocationChangeAction {
```

Comment:
Why do we need to export this interface? This action is supposed to be called internally only.

Reply:
This action will be received in user-defined reducers. Exporting this type allows users to annotate the `action` parameter in their reducers, e.g.: `const reducer = (state, action: LocationChangeAction | MyOtherAction) => state`

Reply:
I need the event as well for user-defined reducer (a reducer where i store the whole history). Is there a way to do it without a LOCATION_CHANGE event ?